src/components/pagination/_pagination.scss

```diff
@@ -219,13 +222,18 @@ $css--helpers: true;
     .#{$prefix}--select-input {
       width: auto;
       min-width: auto;
-      height: auto;
-      padding: 0 1.5rem 0 0.5rem;
-      // TODO: @include type-style('body-short-01');
-      font-size: rem(12px);
-      font-weight: regular;
-      line-height: rem(18px);
-      letter-spacing: 0.16px;
+      height: 100%;
+      padding: 0 2.5rem 0 $spacing-md;
+      margin-right: -0.65rem;
+      @include breakpoint($pagination-breakpoint) {
+        padding: 0 1.5rem 0 0.5rem;
+        margin-right: 0;
+      }
+      @include type-style('body-short-01');
+
+      &:hover {
```

Comment:
Could we un-nest the styles in new code? Totally get that past code is nested but we've been trying to un-nest in the updates to help with searching later down the road 👍